.gitignore

```diff
@@ -14,7 +14,7 @@
 /target
 
 ## Worker.
-/worker/out
+out/
```

Comment:
Why this change? Why was `worker/out` worse? INMHO it makes sense that worker binary is located in `worker/` folder. Do I miss something?

Reply:
I needed a place to download the prebuilds to, I wanted them to be different than `worker/out` so we wouldn't accidentally start taring tarballs, so I put them in `repo-root/out`. the binary is still in `worker/out`. 

`repo-root/out` is fairly arbitrary, I was guessing what you'd prefer based on patterns in the repo, do you have thoughts on a different location for that?

Reply:
I would love worker/prebuilds/ and of course gitignore it.